ApiBundle/Controller/ContentTypeController.php

```diff
@@ -63,6 +63,12 @@ public function listAction(Request $request)
         );
 
         $repository = $this->get('open_orchestra_model.repository.content_type');
+
+        if ($request->get('entityId')) {
```

Comment:
I find this action a little confusing now because there is a return in the middle of it that may not be seen at the first time. Maybe this could be clearer if you introduces two new sub-methods dedicated to the two differents transformations.
The controller code could  be something simple and readable as :
<code>
if ($request->get('entityId')) {
    return $this->transformElement(...);
} else {
    return $this->transformCollection(...);
}
</code>